utilities/transactions/write_unprepared_txn.cc

```diff
@@ -71,8 +70,13 @@ WriteUnpreparedTxn::~WriteUnpreparedTxn() {
 
 void WriteUnpreparedTxn::Initialize(const TransactionOptions& txn_options) {
   PessimisticTransaction::Initialize(txn_options);
-  max_write_batch_size_ = txn_options.max_write_batch_size;
-  write_batch_.SetMaxBytes(0);
```

Comment:
Why is this line deleted?

Reply:
We're setting the max write batch size to infinite here. However, the goal of this change is that we want write_batch_ to have whatever limit is specified in `txn_options.max_write_batch_size`, to have consistent behaviour with other write policies. This happens in `PessimisticTransaction::Initialize`.